sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -204,6 +204,16 @@ export function useFormioDropdownLoader() {
     );
   };
 
+  const loadInstitutionName = async (form: any, dropdownName: string) => {
```

Comment:
Please use `form: FormIOForm` instead of `any`.
The program year using `form: any` was a gap in the previous PR.